samtranslator/internal/schema_source/aws_serverless_graphqlapi.py

```diff
@@ -43,6 +43,18 @@ class DynamoDBDataSource(BaseModel):
     Versioned: Optional[PassThroughProp]
 
 
+class Runtime(BaseModel):
+    Name: str
+    Version: str
+
+
+class ResolverCodeSettings(BaseModel):
```

Comment:
What's the rationale behind choosing between "config", "settings", and "properties", or without adjective?

Reply:
This property is so that the customer can define defaults for properties in all resolvers/functions across the template since they're fairly consistent. `ResolverCode` on its own is not clear enough (too similar to things like `InlineCode` which can be confusing). "Settings" IMO adds extra clarity that its not about the Resolver's code, but how the ResolverCode works with everything else.

I'm not 100% attached to the name, and it's a super quick change either way so I think it's fine to ship as is. We're going to bring it up during our next UX review so we'll likely have a finalized property name at that point. 

Do you have any alternatives that come to mind for you?